content/rs/references/client_references/client_ioredis.md

```diff
@@ -0,0 +1,110 @@
+---
+Title: Using Redis with Node.js (ioredis)
+description:
+weight:
+alwaysopen: false
+categories: ["RS"]
+---
+In order to use Redis with Node.js, you need to install a Node.js Redis client.  In the following sections, we demonstrate the use of [ioredis](https://github.com/luin/ioredis), a community recommended Redis client for Node.js with support for Promises built in.  The other community recommended client for Node.js developers is [node_redis]({{< relref "./client_nodejs" >}}).
+
+Additional Node.js clients for Redis can be found under the [Node.js section](https://redis.io/clients#Node.js) of the [Redis Clients page](https://redis.io/clients).
+
+## Installing ioredis
+
+Complete instructions for installing ioredis can be found in the [README file](https://github.com/luin/ioredis/blob/master/README.md). To install ioredis, issue the following command:
```

Comment:
s/issue the following command/run/

In general we use the term "run" and assume that the reader knows to run the command that follows the semi-colon.